### PR TITLE
DOC: updated install for QIIME 1.9.0

### PR DESCRIPTION
**Pull request by gregcaporaso**
~~NOT READY FOR MERGE~~

I used this for the "install fest" and it all seems to be working ok.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -32,16 +32,16 @@ And on the following packages:
...
+* Install [redis](https://pypi.python.org/pypi/redis/). Do this following the instructions on their website.
 
-```bash
-pip install numpy
-pip install cogent burrito qcli emperor pyzmq
-pip install https://github.com/biocore/qiime/archive/master.zip --no-deps
+* Then, install ``qiita-spots`` and it's python dependencies following as follows:
+
+    ```bash
+    pip install numpy
```
Comment (antgonza) [inline INSTALL.md]: are these spaces needed?

Comment (antgonza) [inline INSTALL.md]: Yup, for nicer formating. Erasing comment.

**Comment by antgonza** [comment]
:+1:

File: INSTALL.md [hunk 2]
```diff
@@ -32,16 +32,16 @@ And on the following packages:
...
-Once you have [PostgreSQL](http://www.postgresql.org/download/) and [redis](https://pypi.python.org/pypi/redis/) installed (follow the instruction on their web site), simply run these commands to install qiita and configure the demo environment, replacing $QIITA_DIR for the path where qiita is installed
-(note that if you are not using Ubuntu you might need to follow the instructions in the next section).
+* Install [PostgreSQL](http://www.postgresql.org/download/) and ensure that your ``$PATH`` environment variable includes the postgres binaries. Do this following the instructions on their website.
+* Install [redis](https://pypi.python.org/pypi/redis/). Do this following the instructions on their website.
 
-```bash
-pip install numpy
-pip install cogent burrito qcli emperor pyzmq
-pip install https://github.com/biocore/qiime/archive/master.zip --no-deps
+* Then, install ``qiita-spots`` and it's python dependencies following as follows:
```
Comment (ElDeveloper) [inline INSTALL.md]: Can you remove "following as"?

File: INSTALL.md [hunk 3]
```diff
@@ -68,17 +68,12 @@ qiita webserver start
...
+If running on OS X you should make sure that the Xcode and the Xcode command line tools are installed.
 
-If you receive the following error, you can ignore this step and continue with the qiita installation:
-```bash
-createuser: creation of new role failed: ERROR:  role "postgres" already exists
-```
+### postgres
 
-**Note**: if you are using Postgres.app under OSX, a database user will already be created for your username. If you want to use this, you will need to use this identity, change the `USER` setting under the `[postgres]` section of your Qiita config file.
+If you are using Postgres.app under OSX, a database user will already be created for your username. If you want to use this, you will need to use this identity, change the `USER` setting under the `[postgres]` section of your Qiita config file.
```
Comment (ElDeveloper) [inline INSTALL.md]: Can you also mention `ADMIN_USER`?

**Comment by ElDeveloper** [comment]
Two minor comments, looks good otherwise.

**Pull request merged**